evals/eval_swe_bench.py

```
@@ -0,0 +1,128 @@
+
+    dataset_name = "SWE_test_data"
+    code_text = dataset[0]["text"][:100000]
+    await cognee.add([code_text], dataset_name)
+    await code_graph_pipeline([dataset_name])
+    graph_engine = await get_graph_engine()
+    with open(graph_engine.filename, "r") as f:
+        graph_str  = f.read()
+    
+    problem_statement = dataset[0]['problem_statement']
+    instructions = (
+        f"I need you to solve this issue by looking at the provided knowledge graph and by "
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)